daprdocs/content/en/operations/hosting/kubernetes/kubernetes-overview.md

```diff
@@ -22,7 +22,7 @@ Read [this guide]({{< ref kubernetes-deploy.md >}}) to learn how to deploy Dapr
 
 ## Adding Dapr to a Kubernetes deployment
 
-Deploying and running a Dapr enabled application into your Kubernetes cluster is a simple as adding a few annotations to the deployment schemes. To give your service an `id` and `port` known to Dapr, turn on tracing through configuration and launch the Dapr sidecar container, you annotate your Kubernetes deployment like this. 
+Deploying and running a Dapr enabled application into your Kubernetes deployment is a simple as adding a few annotations to the deployment schemes. To give your service an `id` and `port` known to Dapr, turn on tracing through configuration and launch the Dapr sidecar container, you annotate your Kubernetes deployment like this. For more information check  [dapr annotations]({{< ref kubernetes-annotations.md >}})
```

Comment:
I think the original version makes more sense. You deploy your app "into a cluster" not "into a deployment". If this should be about the Kubernetes deployment, the first sentence should be reworded (Perhaps "Adding a Dapr enabled application into your Kubernetes deployment is as simple as ...") otherwise can change back to "cluster"

While you are at it, please fix"a simple as" --> "as simple as" (a typo in the original)